subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/executer/GradleExecuter.java

```diff
@@ -198,9 +198,9 @@ public void connect(PipedInputStream snk) throws IOException {
     GradleExecuter withNoExplicitNativeServicesDir();
 
     /**
-     * Disables the rendering of stack traces for deprecation logging.
+     * Enables the rendering of stack traces for deprecation logging.
      */
-    GradleExecuter withFullDeprecationStackTraceDisabled();
+    GradleExecuter withFullDeprecationStackTraceEnabled();
```

Comment:
Did you try out if a test fails if there is a deprecation warning, even if the deprecation stack traces are disabled?

Reply:
I worked with that code, it checks each line if it contains the word "deprecated", it doesn't require the stack trace.

Reply:
All hail undetected merge conflicts :/

Reply:
Merge conflict fixed.

Note that we used to have tests that disabled stack traces in the past, and those worked fine. I only switched over the behavior's default.